### PR TITLE
Move A9 to a 100% roll out

### PR DESCRIPTION
**Pull request by GHaberis**
## What does this change?

This PR rolls A9 out to 100% of the audience by removing the `amazonA9Test` A/B test it previously sat behind.

FYI an A9 feature switch already exists and is behind the `dfpEnv.hbImpl.a9` conditional check.

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/dfp/prepare-a9.js [hunk 1]
```diff
@@ -31,7 +28,7 @@ const setupA9: () => Promise<void> = () =>
         return Promise.resolve();
     });
 
-export const setupA9Once: () => Promise<void> = once(setupA9);
```
Comment (GHaberis) [inline static/src/javascripts/projects/commercial/modules/dfp/prepare-a9.js:34]: I checked and this `export` wasn't necessary so I've removed it.

**Review by ripecosta — CHANGES_REQUESTED** [review]
Can we put A9 init behind a switch? Seems like there's a a switch already created called `a9-header-bidding` but it's not being used.

**Comment by GHaberis** [comment]
> Can we put A9 init behind a switch? Seems like there's a a switch already created called `a9-header-bidding` but it's not being used.

I mentioned above but the value of the A9 feature switch `a9-header-bidding` sits behind the `dfpEnv.hbImpl.a9` conditional check, so we're checking the switch already, just indirectly.

**Comment by ripecosta** [comment]
> > Can we put A9 init behind a switch? Seems like there's a a switch already created called `a9-header-bidding` but it's not being used.
> 
> I mentioned above but the value of the A9 feature switch `a9-header-bidding` sits behind the `dfpEnv.hbImpl.a9` conditional check, so we're checking the switch already, just indirectly.

I see now. I searched the code base for the name of the switch and didn't find it but it seems we get its value differently.

**Review by ripecosta — APPROVED** [review]
👍 Good stuff

**Comment by GHaberis** [comment]
@jeteve I'm just tidying up old branches of mine, is this PR still required or can I close it and delete the branch?

**Comment by mxdvl** [comment]
Hi there!
Should this PR remain open? It's been around for a while and should probably either ship or be closed.
If it needs to remain open, could you please change the base branch to main? See mention above for context.

🧹💨 @GHaberis @ripecosta

**Comment by ioanna0** [comment]
This can be closed as is now obsolete. A9 has been released only to US

**Pull request closed without merging**